src/main/java/com/game/Battleship/models/Cell.java

```diff
@@ -0,0 +1,40 @@
+package com.game.Battleship.models;
+
+public class Cell {
+
+    private int raw;
+    private int column;
+    private String value = " ";
```

Comment:
I don't love the idea of you representing "nothing" (an empty space) with "something" (a blank String). Will come back to this later, nothing to change now.